components/CreateEvent.js

```diff
@@ -26,7 +25,15 @@ class CreateEvent extends React.Component {
 
   constructor(props) {
     super(props);
-    const timezone = momentTimezone.tz.guess();
+
+    let timezone;
+    // fallback for old browsers
+    if (!Intl.DateTimeFormat || !Intl.DateTimeFormat().resolvedOptions) {
+      timezone = 'utc';
+    } else {
+      timezone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+    }
+
```

Comment:
Can't we use `tz.guess` like we do in `HeroEventDetails.js`?

Reply:
Yes we can 🚀